starter-code/server.js

```diff
@@ -24,7 +25,12 @@ app.get('/new', function(request, response) {
 app.get('/articles', function(request, response) {
   // REVIEW: This query will join the data together from our tables and send it back to the client.
   // TODO: Write a SQL query which joins all data from articles and authors tables on the author_id value of each
-  client.query(``)
+  client.query(`
```

Comment:
Good job with this query. Note that when you are grabbing a lot of values, you can use the wildcard selector * to do such, instead of having to list 'articles.article_id, title, body ...'